### ISSUE TITLE
PIP requirements cannot be met on a clean install:

### ISSUE THREAD

**Issue by psudopixel** [retrocomputacion/retrobbs#31]
```
~/Documents/src/retrobbs/ [master] pip3 install -r requirements.txt
Collecting hitherdither@ git+https://www.github.com/hbldh/hitherdither (from -r requirements.txt (line 3))
  Cloning https://www.github.com/hbldh/hitherdither to /private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/hitherdither_c3ead66527dc4115afc84840e8a4a802
  Running command git clone --filter=blob:none --quiet https://www.github.com/hbldh/hitherdither /private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/hitherdither_c3ead66527dc4115afc84840e8a4a802
  warning: redirecting to https://github.com/hbldh/hitherdither.git/
  Resolved https://www.github.com/hbldh/hitherdither to commit b688094bb1e10bf5eda1ba999eb3c98a46ed9407
  Preparing metadata (setup.py) ... done
Collecting pafy@ git+https://github.com/Cupcakus/pafy (from -r requirements.txt (line 12))
  Cloning https://github.com/Cupcakus/pafy to /private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90
  Running command git clone --filter=blob:none --quiet https://github.com/Cupcakus/pafy /private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90
  Resolved https://github.com/Cupcakus/pafy to commit 45f0deb067bf7c420cdf83a0529fd5274c12de18
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [17 lines of output]
      Traceback (most recent call last):
        File "/private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90/pafy/pafy.py", line 48, in <module>
          import youtube_dl
      ModuleNotFoundError: No module named 'youtube_dl'

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90/setup.py", line 13, in <module>
          from pafy import __version__
        File "/private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90/pafy/__init__.py", line 7, in <module>
          from .pafy import new
        File "/private/var/folders/7h/sdb45ws56rq9phtf59l13cy40000gn/T/pip-install-d38t44pg/pafy_e205c64c0ac64b70b75d2146603a4c90/pafy/pafy.py", line 51, in <module>
          raise ImportError(
      ImportError: pafy: youtube-dl not found; you can use the internal backend by setting the environmental variable PAFY_BACKEND to "internal". It is not enabled by default because it is not as well maintained as the youtube-dl backend.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by retrocomputacion** [comment]
Hmm is possible youtube-dl is being removed from the repositories (replaced by yt-dlp). I'll comment out the references to Pafy for now, just in case it restarts development in the future.

**Comment by retrocomputacion** [comment]
Should be fixed in [05976fd](https://github.com/retrocomputacion/retrobbs/commit/05976fda8063a81c751f8f772d094175c16853ea), please confirm

**Comment by psudopixel** [comment]
confirmed.
